tests/test_conformers.py

```diff
@@ -310,3 +310,24 @@ def test_keep_conformers_from_indices_keep_ids():
 
     assert mol2.GetNumConformers() == 3
     assert conf_ids == [0, 4, 6]
+
+
+def test_conformer_energy():
+    mol = dm.to_mol("O=C(C)Oc1ccccc1C(=O)O")
+
+    mol1 = dm.conformers.generate(mol, ewindow=7)
+    assert np.isclose(mol1.GetNumConformers(), 40, atol=20)
```

Comment:
@matudor  `atol` of 20 seems very high. 

I am confident you want to setup some random seed for this process. 

```
random.seed(1)
np.random.seed(1)
```

Then test locally on the computed energy values, then set them up in the test. 
You can limit test to this file for dev.

Reply:
@matudor, tests are green !!

Mind addressing this so I can merge ?

Reply:
good point, I had thought RDkit conformer function set seed but now see that I was wrong.  will work on it.